package.json

```diff
@@ -441,8 +441,17 @@
           "type": "boolean",
           "description": "[Optional] Configuration options for copying multi-line results from the Results View",
           "default": true
+        },
+        "mssql.splitPaneSelection": {
+          "type": "string",
+          "description": "[Optional] Configuration options for which column new result panes should open in (Options: {'next'|'same'|'last'})",
```

Comment:
I think you can remove the `(Options: {'next'|'same'|'last'})` part - the enum values are listed as options by VSCode when altering them